deployment/constructs/dynamodb-table.ts

```diff
@@ -29,7 +29,8 @@ export class DynamoDBTable extends Construct {
   constructor(scope: Construct, id: string, props: DynamoDBTableProps) {
     super(scope, id);
 
-    const tableName = `${props.stage}-${props.name}`;
+    // const tableName = `${props.stage}-${props.name}`;
+    const tableName = `cmdct-4186-sls-${props.name}`;
```

Comment:
This is a tricky part. We have to use the old dynamo names but then keep using them always. And ideally do this in a way that will work for dev branches, though that is probably a nice to have.